sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 4.0.0-beta.5 (Unreleased)
 
 ### Features Added
+- Added `Length` property to `BoundingPolygon`.
```

Comment:
In future please consider adding this to the PR description as well. As an open source repo the more explicit we are the better :)

Reply:
That's a good point. Updated the description of this PR.

Reply:
Awesome thank you :)